Makefile

```diff
@@ -116,7 +116,7 @@ ifeq ($(shell uname), Darwin)
 	easy_install ply
 else
 	apt-get install asciidoc python-dev python-setuptools python-ply
-	apt-get install libgmp3c2
+	apt-get install libgmpxx4ldbl
```

Comment:
libgmpxx4ldbl does not provide libgmp.so.3, which is required by the Bluespec compiler.

libgmp3c2 is no longer available in Ubuntu, so it's not much use having it there.

We currently install libgmp10 and then add a symbolic link:
    ln -s libgmp.so /usr/lib/x86_64-linux-gnu/libgmp.so.3

If you'd like to revise this pull request to do this instead, I would appreciate it.

I highly recommend installing the connectal package in order to install connectal and its dependences, so extra bonus points for creating a libgmp3c2-compatibility debian package, or maybe connectal-libgmp3c2, which creates the link.